CHANGELOG.md

```diff
@@ -5,6 +5,8 @@
   * Migrate more inputs to MUI
   * Fix internet radio searching during stream creation
   * Download appliance config when toggling LMS mode off
+  * Add hoverover effects on player cards
+  * Enforce source selection
```

Comment:
I was uncertain how to word this without being too verbose, I feel that this has the opposite issue of being too low detail to explain "If there is a source that exists, you will always have a source selected"

Reply:
Maybe handle missing source selection. Current wording is fine btw.